docs/operations/data_migration.md

```diff
@@ -28,8 +28,7 @@ information to make connections between registry users and the domains that
 they manage. The registrar stores very few fields about a domain except for
 its name, so it could be straightforward to import the exported list of domains
 from Verisign's `escrow_domains.daily.dotgov.GOV.txt`. It doesn't appear that
-that table stores a flag for active or inactive, so every domain in the file
-can be imported into our system as `is_active=True`.
+that table stores a flag for active or inactive.
```

Comment:
was there a reason to remove the  "is_active=True" part, such as a new requirement?

Reply:
It originally existed on `Domain` because `Domain` has been a stub standing in for something we hadn't developed yet.

I removed it in this PR because I think it will be better to ask the registry rather than caching `is_active` in the registrar database. If it turns out to be too much of a performance cost, you could bring it back, I think.

In terms of performance vs stability, I have been favoring stability in my design choices.

Reply:
I definitely agree with favoring stability as well and like that approach. Not marking them as active by default also checks out for me, thanks for explaining!